lib/DocumentFilter/DocumentFilterRule.test.js

```diff
@@ -51,52 +54,65 @@ describe('DocumentFilterRule', () => {
     await Button('Submit').exists();
   });
 
-  it('renders Name option', () => {
+  test('renders the correct options for the Attribute dropdown', async () => {
     TextArea('Name').exists();
-  });
-
-  it('renders Note option', () => {
     TextArea('Note').exists();
-  });
-
-  it('renders Physical location option', () => {
+    TextArea('Category').exists();
     TextArea('Physical location').exists();
-  });
-
-  // TODO this is not how we check select options
-  it('renders URL option', () => {
     TextArea('URL').exists();
-  });
-
-  it('renders Content type option', () => {
     TextArea('Content type').exists();
-  });
-
-  it('renders File name option', () => {
     TextArea('File name').exists();
   });
 
-  it('renders Category option', () => {
-    TextArea('Category').exists();
-  });
-
-  it('renders is option', () => {
+  test('renders the correct options for the Comparator dropdown', async () => {
     TextArea('is').exists();
-  });
-
-  it('renders containes option', () => {
     TextArea('containes').exists();
-  });
-
-  it('renders does not contain option', () => {
     TextArea('does not contain').exists();
   });
 
-  // TODO check Delete button renders for index > 0 and calls onDelete as expected and does NOT render for index 0
-  // Test that value is normally a TextField
-  // Test that selecting category makes:
-  // the comparator a Select with is vs is not
-  // the value a Select with expected options (categoryvalues from some centralised test resources)
 
-  // Also please test DocumentFilterRuleConstants returns what you expect for ALL occasions
+  describe('DocumentFilterRule with more than one rule', () => {
+    beforeEach(() => {
+      renderWithIntl(
+        <MemoryRouter>
+          <TestForm onSubmit={onSubmit}>
+            <Field
+              name="filters[0].rules[1]"
+            >
+              {({ input: { name } }) => (
+                <DocumentFilterRule
+                  ariaLabelledby="selected-document-item-name-0"
+                  categoryValues={categoryValues}
+                  index={1}
+                  name={name}
+                  onDelete={onDelete}
+                  value={value}
+                />
+              )}
+            </Field>
+          </TestForm>
+          ,
+        </MemoryRouter>,
+        translationsProperties
+      );
+    });
+
+    test('renders the submit button', async () => {
+      await Button('Submit').exists();
+    });
+
+    test('removing rules using the remove button works as expected', async () => {
+      const { getByRole } = renderComponent;
+      expect(getByRole('button', { name: /remove rule 2/i })).toBeInTheDocument();
```

Comment:
Why is this not using Button interactor? Same as above, if there's a reason there needs to be a comment